user-guide/Getting_started/Installing_a_DMA/Installing_DM_using_the_DM_Installer.md

```diff
@@ -33,128 +33,41 @@ The DataMiner installer allows you to run a default DataMiner installation, whic
 1. Download the DataMiner installer from [DataMiner Dojo](https://community.dataminer.services/download/dataminer-installer-v10-2).
```

Comment:
This link will need to be updated to the 10.4 installer before we can merge this.

Reply:
Can you link to https://community.dataminer.services/dataminer-installer/ instead? Then the link doesn't need to be updated on every installer update. We will be releasing new installer versions more frequently in the future, linked to the DataMiner version.